### PR TITLE
Updates for Production / Dual Repo System

### PR DESCRIPTION
**Pull request by shikokuchuo**
I've made a start on this.

Concentrates on the home page and about page for now.

- Simplifies homepage - I want to keep this as clean as possible
- Splits About into About -> Community  | Production
- Updates User instructions to include both repos

I've focused on the logical flow and getting the text clear and concise.

Suggestions on graphics etc. welcome (direct commits even more so!).

### CODE REVIEW

**Comment by shikokuchuo** [comment]
Actually, given where we are, I suggest to merge this or a version with the minimum changes needed first, and then open up new issues for improvements.

As our new snapshot plan hasn't made it onto the website yet at this point.

File: _quarto.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ project:
 
 website:
   title: "R-multiverse"
-  description: "A community-curated collection of R package releases, powered by R-universe."
+  description: "A dual package repository system, powered by R-universe"
```
Comment (maelle) [inline _quarto.yml:10]:
```suggestion
  description: "A dual package repository system, powered by R-universe and curated by the community"
```

or something else allowing to state it's community-curated

File: community.qmd [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## An Open Repository for R Packages
+
+Packages are available for installation from the repository at <https://community.r-multiverse.org>.
+
+A traditional software version cycle begins with the development phase, where bugs are identified and fixed, and ends with the production phase, which promises a seamless user experience.
+
+Community aims at a space just before production, commonly referred to as Quality Assurance (QA). This means that:
+
+1. Each package release has the full endorsement of its own maintainer. R-multiverse always provides a version that its developer chose to distribute for general use.
+2. The user is still responsible for judging whether a package is appropriate to use. Some releases may still have issues, as indicated by its check results, publicly available at <https://community.r-multiverse.org>.
```
Comment (maelle) [inline community.qmd:14]:
```suggestion
2. The user is still responsible for judging whether a package is appropriate to use. Some releases may still have issues, as indicated by their check results, publicly available at <https://community.r-multiverse.org>.
```

File: community.qmd [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Continuous maintainer-driven deployment
+
+Anyone can [contribute](contributors.qmd) packages to Community via GitHub. Each package only needs to be registered once.
```
Comment (maelle) [inline community.qmd:18]: line breaks after sentences, I promise it makes future diffs worth it :smile_cat:

File: community.qmd [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Every new [release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) then automatically deploys to the repository without needing to go through R-multiverse again. In other words, the package maintainer is in complete control.
+
+This frictionless, maintainer-driven experience was made possible by the incredible infrastructure of rOpenSci's R-universe system.
```
Comment (maelle) [inline community.qmd:22]:
```suggestion
This frictionless, maintainer-driven experience was made possible by the incredible infrastructure of [rOpenSci's R-universe system](https://ropensci.org/r-universe/).
```

File: index.qmd [hunk 5]
```diff
@@ -4,8 +4,19 @@ title: "R-multiverse"
...
 <img src="logo/logo-hexbin.png" align="right" width="140" />
 
-A community-curated collection of R package releases, powered by [R-universe](https://r-universe.dev), available for installation from the repository at <https://community.r-multiverse.org>.
+#### A dual package repository system, powered by R-universe
 
-Anyone can contribute packages to the GitHub repository at <https://github.com/r-multiverse/contributions>. Once a package is registered, every new release is automatically built and checked with results available at <https://community.r-multiverse.org>. Package maintainers retain complete control over deployment.
+#### [Community](community.qmd)
 
-The R-multiverse project originated from the [R Consortium Repositories Working Group](https://github.com/RConsortium/r-repositories-wg) and is supported by an [R Consortium](https://www.r-consortium.org/) ISC grant.
+An open and inclusive R package repository with transparent governance standards, built on open source infrastructure.
```
Comment (maelle) [inline index.qmd:11]: add link in text again in case the reader does not think of clicking on the title

File: index.qmd [hunk 6]
```diff
@@ -4,8 +4,19 @@ title: "R-multiverse"
...
+
+#### +
```
Comment (maelle) [inline index.qmd:13]: ?

File: index.qmd [hunk 7]
```diff
@@ -4,8 +4,19 @@ title: "R-multiverse"
...
+
+#### [Production](production.qmd)
+
+A quarterly snapshot release of R packages, designed for deployment in production environments.
```
Comment (maelle) [inline index.qmd:17]: add link in text

File: index.qmd [hunk 8]
```diff
@@ -4,8 +4,19 @@ title: "R-multiverse"
...
+
+<br />
```
Comment (maelle) [inline index.qmd:19]: why not add this as a footer on all pages thus controlled by the Quarto config file? I think that's possible

File: production.qmd [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+But we take it one step further.
+
+Instead of the continuous rolling deployment model that CRAN provides, we provide snapshots of fixed package versions. This has the advantage that:
+
+1. All packages within a snapshot are guaranteed to work with each other.
+ + At any one point in time on CRAN, there will be packages that fail checks but are within grace periods and have not yet been updated or archived.
+2. Using fixed package versions aids reproducibility.
+3. Suited to environments without the ability to update over the internet.
+
+We take quarterly snapshots on the 15th of each month in May, August, November and February.
```
Comment (maelle) [inline production.qmd:24]: genuine question. should people count on the snapshots to be maintained forever or should they download them.

File: production.qmd [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+## The Production Staging Process
+
+Before each quarterly release, packages go through a one-month 'staging' process, which helps ensure packages continue to meet the high standards required of Production.
+
+Packages enter into the Staging repository at <https://staging.r-multiverse.org/>. For those with passing checks, the package version is frozen.
```
Comment (maelle) [inline production.qmd:30]: what's the "3" version of dual :stuck_out_tongue_winking_eye:

File: production.qmd [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+
+- Even if another package release is made and Community updates, the version in Staging remains the same.
+
+Packages with failing checks, however, will continue to update from Community if a new release is made.
+
+This process helps maintainers to fix bugs, whilst minimizing the chances of last minute issues affecting large cohorts of packages.
+
+Staging opens on the 15th of each month in April, July, October and January.
```
Comment (maelle) [inline production.qmd:38]: starting which year-month

File: production.qmd [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+- Windows: R-4.4 binary, R-4.3 binary
+- Mac x86: R-4.4 binary, R-4.3 binary
+- Mac arm64: R-4.4 binary, R-4.3 binary
+
+It may be used in R repository settings or directly as the 'repos' argument to `install.packages()`. To query available packages:
+
+```r
+available.packages(repos = "https://production.r-multiverse.org")
+```
+*[Our first Production release is scheduled for 15 November 2024, and prior to that, a sample repository is available at <https://r-multiverse.org/snapshot>]*
```
Comment (maelle) [inline production.qmd:56]: oh I see it's here, maybe put it earlier near the months

File: users.qmd [hunk 13]
```diff
@@ -4,15 +4,19 @@ title: "Users"
 
 ## How to install a package from R-multiverse
 
-R has built-in support for multiple repositories.
+R has built-in support for multiple repositories. To add R-multiverse to your repository settings, a line can be added to your `.Rprofile` as follows.
```
Comment (maelle) [inline users.qmd:7]:
```suggestion
R has built-in support for multiple repositories. To add R-multiverse to your repository settings, a line can be added to your [`.Rprofile`](https://rstats.wtf/r-startup.html#rprofile) as follows.
```

**Review by maelle — APPROVED** [review]
Thank you!!

File: _quarto.yml (hunk 1 above)
Comment (shikokuchuo) [reply _quarto.yml:10]: "A community-curated package repository system, powered by R-universe"

The fact it's 'dual' is not integral to its purpose. In fact we also have a 3rd staging repo as you commented below....

File: community.qmd (hunk 3 above)
Comment (shikokuchuo) [reply community.qmd:18]: Thanks! I will definitely get into this habit. Have made changes elsewhere as well to implement this.

File: index.qmd (hunk 5 above)
Comment (shikokuchuo) [reply index.qmd:11]: I've taken the links out from the titles and put them underneath. Looks nicer as well I think.

File: production.qmd (hunk 9 above)
Comment (shikokuchuo) [reply production.qmd:24]: I think this is still an open question, whether we will maintain an archive. So there is no answer yet (at least for the website).

File: production.qmd (hunk 10 above)
Comment (shikokuchuo) [reply production.qmd:30]: I've taken out mentions of 'dual' as it's not important!

File: production.qmd (hunk 12 above)
Comment (shikokuchuo) [reply production.qmd:56]: I've copied this above to where the months are first mentioned.

File: index.qmd (hunk 7 above)
Comment (shikokuchuo) [reply index.qmd:17]: done here as well

File: index.qmd (hunk 6 above)
Comment (shikokuchuo) [reply index.qmd:13]: Think this is just the GitHub diff causing confusion, this text is still retained later.

File: index.qmd (hunk 8 above)
Comment (shikokuchuo) [reply index.qmd:19]: If you're talking about the R-consortium support, I did try and it ends up being too long and too ugly so I left it only on the homepage.

**Comment by shikokuchuo** [comment]
Thanks @maelle for the review. I've incorporated all of your comments.

I'll merge this now and if anyone spots anything, do comment further and I'll make the changes.

File: index.qmd (hunk 8 above)
Comment (maelle) [reply index.qmd:23]: @shikokuchuo just remembered we have such a footer on https://r-dbi.org/ Is it ugly :stuck_out_tongue_winking_eye: (it's short)

Comment (shikokuchuo) [reply index.qmd:23]: No that looks good. So how about we just add "Supported by the R Consortium" in the footer? It was ugly when I tried to include the whole blurb about the working group...

Comment (maelle) [reply index.qmd:23]: Yes it'd be a good idea I think, and you could keep the blurb in about.

**Pull request merged**